### PR TITLE
[feat][doc] Add doc for the HTTP Sink

### PR DESCRIPTION
**Pull request by cbornet**
### Documentation

- [x] `doc` 
- [ ] `doc-required` 
- [ ] `doc-not-needed` 
- [ ] `doc-complete` 

### Matching PR in forked repository

PR in forked repository: https://github.com/cbornet/pulsar/pull/10

### CODE REVIEW

**Comment by Anonymitaet** [comment]
Thanks for your contribution! Writers will review this PR from technical writing pespectives after it gets technical reviews from engineers.

File: site2/docs/io-http-sink.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
+---
+id: io-http-sink
+title: HTTTP sink connector
+sidebar_label: "HTTP sink connector"
+---
+
+The HTTP sink connector pulls the records from Pulsar topics and makes a POST request to a configurable HTTP url (webhook).
```
Comment (momo-jun) [inline site2/docs/io-http-sink.md:7]:
```suggestion
The HTTP sink connector pulls the records from Pulsar topics and makes a POST request to a configurable HTTP URL (webhook).
```

File: site2/docs/io-http-sink.md [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+## Configuration
+
+The configuration of the HTTP sink connector has the following properties.
+
+### Property
+
+| Name      | Type   | Required | Default          | Description |
+|-----------|--------|----------|------------------|-------------|
+| `url`     | String | false    | http://localhost | The URL of the HTTP server |
+| `headers` | Map    | false    | empty map        |The HDFS directory where files read from or written to. |
```
Comment (momo-jun) [inline site2/docs/io-http-sink.md:29]:
```suggestion
| `headers`             | Map         | false    | empty map        |The HDFS directory where files are read from or written to. |
```

Comment (momo-jun) [inline site2/docs/io-http-sink.md:29]:
```suggestion
| `headers` | Map    | false    | empty map        |The HDFS directory where files are read from or written to. |
```

**Comment by momo-jun** [comment]
@cbornet thanks for adding the docs. It deserves a `feature` in the PR title.

File: site2/docs/io-http-sink.md (hunk 2 above)
Comment (cbornet) [reply site2/docs/io-http-sink.md]: The description was an incorrect copy/paste. I fixed it, PTAL.

File: site2/docs/io-http-sink.md [hunk 3]
```diff
@@ -0,0 +1,56 @@
+---
+id: io-http-sink
+title: HTTTP sink connector
+sidebar_label: "HTTP sink connector"
```
Comment (tisonkun) [inline site2/docs/io-http-sink.md:4]: I think we don't show this page on the sidebar so this metadata is unnecessary. But it's not a blocker.

**Review by tisonkun — APPROVED** [review]
LGTM.

File: site2/docs/io-http-sink.md (hunk 3 above)
Comment (cbornet) [reply site2/docs/io-http-sink.md:4]: Then it should be removed from all the connector pages I guess ?

Comment (tisonkun) [reply site2/docs/io-http-sink.md:4]: Yep. I comment here, and possibly we file an issue to track such a movement :)

**Review by Anonymitaet — APPROVED** [review]

File: site2/docs/io-http-sink.md [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+The body of the HTTP request is the JSON representation of the record value. The header `Content-Type: application/json` is added to the HTTP request.
+
+Some other HTTP headers are added to the HTTP request:
+
+* `PulsarTopic`: the topic of the record
+* `PulsarKey`: the key of the record
+* `PulsarEventTime`: the event time of the record
+* `PulsarPublishTime`: the publish time of the record
+* `PulsarMessageId`: the ID of the message contained in the record
+* `PulsarProperties`-*: each record property is passed with the property name prefixed by `PulsarProperties-`
```
[the remark this answers is not in the document]
Comment (momo-jun) [reply site2/docs/io-http-sink.md:18]: `PulsarProperties-*` +1. Better to include the wildcard in the quotes.

[the remark this answers is not in the document]
Comment (cbornet) [reply site2/docs/io-http-sink.md]: Oups, I committed the wrong suggestion. I did another commit to adjust.

**Comment by tisonkun** [comment]
Merging...

**Pull request merged**